tutorials/io/plot_20_reading_eeg_data.py

```diff
@@ -211,15 +243,16 @@
 the data are assumed to already be properly referenced. See
 :ref:`tut-set-eeg-ref` for more information.
 
+
 Reading electrode locations and head shapes for EEG recordings
 ==============================================================
 
-Some EEG formats (EGI, EDF/EDF+, BDF) neither contain electrode location
-information nor head shape digitization information. Therefore, this
-information has to be provided separately. For that purpose all raw instances
-have a :meth:`mne.io.Raw.set_montage` method to set electrode locations.
+Some EEG formats (EGI, EDF/EDF+, BDF) contain neither electrode locations nor
```

Comment:
I think there are more, so let's make sure this is an example, not an exhausitve list.

For instance, I have never seen a BrainVision file with **digitized** coordinates embedded. It's possible (see https://github.com/bids-standard/pybv/issues/44), but I have never seen it - have you? Usually **if** there are coords at all, they are idealized template locations.

```suggestion
Some EEG formats (e.g., EGI, EDF/EDF+, BDF) contain neither electrode locations nor
```